image/base/install-oxipng

```diff
@@ -2,23 +2,24 @@
 set -e
 
 # version check: https://github.com/shssoichiro/oxipng/releases
-OXIPNG_VERSION="8.0.0"
-OXIPNG_HASH="ef96d6340e70900de0a38ace8f5f20878f6c256b18b0c59cd87f2b515437b87b"
-OXIPNG_ARCHIVE="v${OXIPNG_VERSION}.tar.gz"
-OXIPNG_DIR="oxipng-${OXIPNG_VERSION}"
+OXIPNG_VERSION="9.1.2"
+dpkgArch="$(dpkg --print-architecture)"
 
-# Install other deps
-apt-get -y install advancecomp jhead jpegoptim libjpeg-turbo-progs optipng
-
-cd /tmp
-wget -q https://github.com/shssoichiro/oxipng/archive/refs/tags/${OXIPNG_ARCHIVE}
-sha256sum ${OXIPNG_ARCHIVE}
-echo "${OXIPNG_HASH} ${OXIPNG_ARCHIVE}" | sha256sum -c
+case "${dpkgArch##*-}" in
+    amd64) OXIPNG_FILE="oxipng-${OXIPNG_VERSION}-x86_64-unknown-linux-musl.tar.gz"; OXIPNG_HASH='211d53f3781be4a71566fbaad6611a3da018ac9b22d500651b091c2b42ebe318' ;;
+    arm64) OXIPNG_FILE="oxipng-${OXIPNG_VERSION}-aarch64-unknown-linux-musl.tar.gz"; OXIPNG_HASH='818d47d7195e1e0c4d58a9f3b6fd84aa3cd21770c60c876e73e2e6a17ca69b52' ;;
+    *) echo >&2 "unsupported architecture: ${dpkgArch}"; exit 1 ;;
+esac
```

Comment:
OxiPNG started shipping releases for linux arm64 so we no longer have to compile this thus removing the need for us to install Rust in the base image.